README.md

```diff
@@ -21,9 +21,14 @@ You can use the resulting tokens with any service that supports JWT tokens.
 To run the service you need to create the configuration that contains all the needed keys and identifiers.
 To do this, generate a fresh DID for the service deployment and then create a config file from it.
 
+First assign your KILT account seed to a variable named SEED then run opendid-setup container.
+
+`SEED="dont try this seed its completely made up for this nice example"`
+
+If your account is on KILT Spiritnet, ensure that you include `ENDPOINT=spiritnet` environment variable; otherwise, the command will automatically default to Peregrine testnet chain.
```

Comment:
Is this what we want? Do we want to have a default at all? Was this already decided? I would not use a default or, if there needs to be a default, I would make it Spiritnet and not Peregrine.

Reply:
I had the same intuition; I think it makes sense to force users to choose for which network to run the setup. And for running the actual login container, I would even force users to supply the URL of the full node (or nodes) they want to connect to, as this, I my view, should be a conscious choice. If we give concrete values they may want to use in the README and/or an example env file, this shouldn't be a huge hurdle either.

Reply:
I am also in favor of having no default value.

Reply:
Removed default

Reply:
I see also some scripts assume Peregrine is the default. If those are used in the demo, they should also be changed to not assume any defaults, e.g., for `gen-did` script. Please check those occurrences as well.